### ISSUE TITLE
Decide on CQL LOGGED / UNLOGGED BATCH for deleteMany and updateMany 

### ISSUE THREAD

**Issue by amorton** [stargate/jsonapi#146]
We are assuming the full support for CQL batching [here](https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/cqlBatch.html) and variation due to the bridge is not part of the discussion. 

Background reading see reasons for using a batch [here](https://docs.datastax.com/en/cql-oss/3.x/cql/cql_using/useBatch.html)

deleteMany and updateMany will be doing multi partition updates that are protected by LWT. 

We de done need to run UNLOGGED batches which is often used to ensure statements on the same partition or coalesced into a single internal mutation. 

A reason for using LOGGED batch would be if all the updates are correlated in some way such a referential integrity across tables or partitions. We do not have any concerns for that, the mutations are independant. 

Finally, failure modes. Running a LOGGED batch would ensure the operations complete if the coordinator or APi server failed. In the case of coordinator failure we would not be able to provide a decent service to our clients, that is we could not say what had been applied or would be applied later. In the case of failure of our service we again would not be able to say what had applied. So there is no benefit from using a LOGGED batch, and there is a non trivial performance hit on coordinators (1 extra round trip and memory usage).

So we should send multiple independent commands. 

QUESTIon - did we decide if updateMany and deleteMany were sequential fail fast operations ?

**Comment by ivansenic** [comment]
* As I understand, we agree to not use Cassandra batches here (neither logged nor unlogged), but individual statements. :heavy_check_mark: 
* Sequential execution makes those operations very slow, basically hitting N+1 query issue. In addition, it provides no better feedback to the user at the end. If we decide to limit amount of many operations executed in one hit, then there is no reason not to do all of those ops in parallel and report results back. :question:

**Comment by ivansenic** [comment]
Closing. We are issuing a request per delete.
